src/promptflow-core/promptflow/executor/flow_validator.py

```diff
@@ -182,7 +182,9 @@ def resolve_aggregated_flow_inputs_type(flow: Flow, inputs: Mapping[str, List[An
         return updated_inputs
 
     @staticmethod
-    def resolve_flow_inputs_type(flow: Flow, inputs: Mapping[str, Any], idx: Optional[int] = None) -> Mapping[str, Any]:
```

Comment:
Do not directly change the interface of a public function, we may add another one.

Reply:
Refined.